DESCRIPTION

```diff
@@ -1,15 +1,37 @@
 Package: xMSannotator
```

Comment:
should we make this similar to recetox.aplcms here: https://github.com/RECETOX/recetox-aplcms/blob/master/DESCRIPTION#L1